CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 0.4.0 / 2017-10-02
 ==================
 
+  * Added context menus to the application
```

Comment:
From now on, don't change the changelog (no problem here, I'll change it myself) as there are conflicts among contributors